ros_ws/src/control/src/control/remote_control.cpp

```diff
@@ -17,7 +17,7 @@ RemoteControl::RemoteControl()
     cmd_vel.angular.y = 0;
     cmd_vel.angular.z = 0;
     speed_modifier = 0.5;
-
+    config = initConfigs();
```

Comment:
Does it need to be added to  RemoteControl::RemoteControl(std::string twist_topic, std::string joy_topic) ?